### DISCUSSION TITLE
Welcome to f1tenth_system Discussions!

### DISCUSSION THREAD

**Discussion by rahulmangharam** [f1tenth/f1tenth_system#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CrossWax** [comment]
Any suggestions on the throttle interpolate, is it ready to use. I saw that in the launch file, it is commented out. 
Thanks
